DataCollector/RulerDataCollector.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Hoathis\Bundle\RulerBundle\DataCollector;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface;
+
+use Hoathis\Bundle\RulerBundle\Ruler\LoggedRuler;
```

Comment:
I have created an issue for the bench bundle: https://github.com/hoaproject/Contributions-Symfony-BenchBundle/issues/5.